client.js

```diff
@@ -26,7 +26,9 @@ function DHT (opts) {
   this._peers = new PeerStore(opts.maxPeers || 10000)
 
   this._secrets = null
-  this._rpc = krpc(opts)
+  this._hash = opts.hash || sha1
+  this._hash_length = this._hash(Buffer.from('')).length
```

Comment:
We use `camelCase` in this project. The leading underscore only indicates a private variable. Can you rename this to `this._hashLength`?